services/core-web/src/components/mine/Variances/MineVarianceTable.js

```diff
@@ -59,48 +58,47 @@ const hideColumn = (condition) => (condition ? "column-hide" : "");
 const applySortIndicator = (_columns, field, dir) =>
   _columns.map((column) => ({
     ...column,
-    sortOrder: column.sortField === field ? dir.concat("end") : false,
+    sortOrder: dir && column.sortField === field ? dir.concat("end") : false,
   }));
 
 const handleTableChange = (updateVarianceList) => (pagination, filters, sorter) => {
-  const params = isEmpty(sorter)
-    ? {
-        sort_field: undefined,
-        sort_dir: undefined,
-      }
-    : {
-        sort_field: sorter.column.sortField,
-        sort_dir: sorter.order.replace("end", ""),
-      };
+  const params = {
+    results: pagination.pageSize,
+    page: pagination.current,
+    sort_field: sorter.field,
+    sort_dir: sorter.order ? sorter.order.replace("end", "") : sorter.order,
+    ...filters,
+  };
   updateVarianceList(params);
 };
 
 export class MineVarianceTable extends Component {
-  transformRowData = (variances, codeHash, statusHash) =>
+  transformRowData = (variances) =>
     variances.map((variance) => ({
       key: variance.variance_guid,
       variance,
-      mineName: variance.mine_name || Strings.EMPTY_FIELD,
-      mineGuid: variance.mine_guid,
-      status: statusHash[variance.variance_application_status_code],
-      compliance_article_id: codeHash[variance.compliance_article_id] || Strings.EMPTY_FIELD,
+      mine_name: variance.mine_name || Strings.EMPTY_FIELD,
+      mine_guid: variance.mine_guid,
+      variance_application_status_code: variance.variance_application_status_code,
```

Comment:
Is this showing the 3 digit variance code now? should be showing the description

Reply:
disregard